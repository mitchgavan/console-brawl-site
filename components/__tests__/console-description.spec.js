import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ConsoleDescription from '../console-list/console-description';

test('<ConsoleDescription /> renders correctly', () => {
  const tree = shallow(
    <ConsoleDescription
      description="A cool console"
      developer="Nintendo"
      name="Gamecube"
      released="September 14, 2001"
      reverseOrder={false}
    />,
  );
  expect(toJson(tree)).toMatchSnapshot();
});
