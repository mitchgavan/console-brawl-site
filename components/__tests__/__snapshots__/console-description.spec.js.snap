// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConsoleDescription /> renders correctly 1`] = `
<div
  className="root"
>
  <header
    className="header false"
  >
    <h3
      className="title"
    >
      Gamecube
    </h3>
    <div
      className="subtitle"
    >
      Release Date: 
      <strong>
        September 14, 2001
      </strong>
    </div>
    <div
      className="subtitle"
    >
      Developer: 
      <strong>
        Nintendo
      </strong>
    </div>
  </header>
  <div
    className="description description-reverse"
  >
    <p>
      A cool console
    </p>
  </div>
</div>
`;
